components/common/Link.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export const defaultAllowedSchemes = ['http:', 'https:']
```

Comment:
@fmarier @thypon not sure if I should be allowing `http:` here or not - this is for links to other pages, so I feel like it's probably acceptable?

Reply:
Yes, both HTTP and HTTPS are acceptable since we don't control what webpages use, and we have automatic HTTPS upgrades in Brave.